src/components/calcite-alert/calcite-alert.e2e.ts

```diff
@@ -63,10 +79,8 @@ describe("calcite-alert", () => {
     expect(close).not.toBeNull();
     expect(icon).not.toBeNull();
   });
-});
 
-describe("calcite-alert", () => {
-  it("validates incorrect props with dismiss", async () => {
+  it("validates incorrect props and forces close button with dismiss and alert-link present", async () => {
     const page = await newE2EPage();
     await page.setContent(`
     <calcite-alert color="zip" duration="zot" theme="zat" dismiss>
```

Comment:
Might be worthwhile to add a comment that zip zot and zat are invalid values and why.